src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -66,11 +80,29 @@ public DissolutionCreateResponse submitDissolutionRequest(
             throw new UnauthorisedException();
         }
 
+        final CompanyProfileApi companyProfileApi;
+
+        try {
```

Comment:
hmmm now that ServiceException is a RuntimeException I wonder if we need these try catches, or should we let them bubble up into the global error handler which catches all RuntimeExceptions and returns a 500.